src/ServiceManagement/Network/Commands.Network/Gateway/NewAzureVirtualNetworkGatewayConnection.cs

```diff
@@ -32,8 +32,9 @@ public class NewAzureVirtualNetworkGatewayConnectionCommand : NetworkCmdletBase
         [ValidateNotNullOrEmpty]
         public string GatewayConnectionName { get; set; }
 
-        [Parameter(Position = 2, Mandatory = true, HelpMessage = "Gateway connection type: Ipsec/Dedicated/VpnClient/Vnet2Vnet")]
+        [Parameter(Position = 2, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "Gateway connection type.")]
         [ValidateNotNullOrEmpty]
+        [ValidateSet(ServiceManagement.Model.VirtualNetworkGatewayConnectionType.ExpressRoute, ServiceManagement.Model.VirtualNetworkGatewayConnectionType.IPsec, ServiceManagement.Model.VirtualNetworkGatewayConnectionType.Vnet2Vnet, ServiceManagement.Model.VirtualNetworkGatewayConnectionType.VPNClient, IgnoreCase = true)]
```

Comment:
This commandlet would have never worked with any values other than the listed ones in validateset. It used to fail from server side and show general error message as :- "The specified request content was not valid and could not be deserialized" which adds more confusion to customers. So, I don't think this is any breaking change, it will help customers to understand error sooner and in better way.